org.gimp.GIMP.json

```diff
@@ -16,12 +16,34 @@
                     "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*",
                     "--talk-name=org.freedesktop.FileManager1"],
     "tags": ["stable"],
-    "cleanup": ["/include", "/lib/pkgconfig", "/lib/cmake/", "/share/pkgconfig",
-                "/share/aclocal", "/man", "/share/man", "/share/gtk-doc", "/share/doc",
-                "/share/vala", "*.la", "*.a", "/bin/wmf*", "/bin/libwmf-*",
-                "/share/pygobject/2.0/codegen",
-                "/bin/pygtk*", "/bin/pygobject*", "/lib/pygtk",
-                "/bin/python*-config", "/bin/pip*", "/bin/smtpd.py" ],
+    "cleanup": [
+        "/lib/cmake/",
+        "/man", "/share/man",
+        "/share/gtk-doc",
+        "/share/doc",
+        "/share/vala",
+        "*.la",
+        "*.a",
+        "/bin/wmf*",
+        "/bin/libwmf-*",
+        "/share/pygobject/2.0/codegen",
+        "/bin/pygtk*",
+        "/bin/pygobject*",
+        "/lib/pygtk",
+        "/bin/python*-config",
+        "/bin/pip*",
+        "/bin/smtpd.py"
+    ],
+    "add-extensions": {
+        "org.gimp.GIMP.Plugin": {
+            "version": "2-3.36",
+            "directory": "extensions",
```

Comment:
Not exactly. This is the directory in which every extension will have its directory (mounted).

It is the dirctories in `merge-dirs` that will contain (via symlink) all the plug-ins. And now I realize we might need more than just `plug-ins`. `scripts` for example I have seen.

So in the current case, plugins are in `/app/extensions/plug-ins`.
Here is in my local setup the content:
```
$ ls -l /app/extensions/plug-ins/
total 0
lrwxrwxrwx 1 hub hub 34 Jun 13 11:31 bimp -> /app/extensions/BIMP/plug-ins/bimp
lrwxrwxrwx 1 hub hub 54 Jun 13 11:31 gimp-lqr-plugin -> /app/extensions/LiquidRescale/plug-ins/gimp-lqr-plugin
lrwxrwxrwx 1 hub hub 42 Jun 13 11:31 gmic_gimp_qt -> /app/extensions/GMic/plug-ins/gmic_gimp_qt
lrwxrwxrwx 1 hub hub 55 Jun 13 11:31 plug_in_lqr_iter -> /app/extensions/LiquidRescale/plug-ins/plug_in_lqr_iter
lrwxrwxrwx 1 hub hub 52 Jun 13 11:31 resynthesizer -> /app/extensions/Resynthesizer/plug-ins/resynthesizer
lrwxrwxrwx 1 hub hub 56 Jun 13 11:31 resynthesizer_gui -> /app/extensions/Resynthesizer/plug-ins/resynthesizer_gui
```

Reply:
Ah ok so manuals for instance would be in `/app/extensions/help/`? So this `directory` field is generic for all types of extensions?

Why I thought it was specific to the plug-in extension point is because it is under the `"org.gimp.GIMP.Plugin"` dictionnary.

Reply:
If I add `help` to the `merge-dirs` (I will).